src/test/java/fi/otavanopisto/kuntaapi/test/server/integration/ptv/ServiceChannelsTestsIT.java

```diff
@@ -328,6 +328,13 @@ public void testFindServiceLocationChannel() throws InterruptedException {
       .get("/serviceLocationServiceChannels")
       .body().jsonPath().getString("id[0]");
     
+    System.out.println(
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)